### ISSUE TITLE
Update S205 response 

### ISSUE THREAD

**Issue by awolfe76** [cfpb/hmda-platform#1327]
![s205](https://user-images.githubusercontent.com/2932572/32732582-0a7e40fc-c85b-11e7-854b-db3caa64538e.png)

The response looks like

```json
{
  "count": 1,
  "total": 1,
  "edit": "S205",
  "rows": [
    {
      "row": { "rowId": "0000000000" },
      "fields": { "Loan/Application Number": "error: field name mismatch" }
    }
  ],
  "_links": {
    "self": "?page=1",
    "prev": "?page=1",
    "last": "?page=1",
    "next": "?page=1",
    "first": "?page=1",
    "href": "/institutions/6999998/filings/2017/submissions/4/edits/S205{rel}"
  }
}

```

The `rowId` is good here, but the `error: field name mismatch` doesn't seem to make sense. 

Could we update the `fields` to be something like

```json
"rows": [
    {
      "row": { "rowId": "0000000000" },
      "fields": { "Application/Loan Number": "Application/Loan number not in valid format or is missing" }
    }
  ]
```

_New message is from [FIG](https://www.consumerfinance.gov/data-research/hmda/static/for-filers/2017/2017-HMDA-FIG.pdf)._

**Comment by schbetsy** [comment]
Good catch on this. The pattern we've used for each edit so far is "row" contains the Loan/Application Number, then "fields" contains all of the other relevant fields in the lar (where the field name is the json key, and the value it contains in the LAR as the json value). `S205` is weird because the Loan/Application Number is the only relevant field. There's no other part of the LAR that would be helpful to show.

So, if we want to follow the pattern we've used for the other edits, I think we should display it like this: 
<img width="823" alt="screen shot 2017-11-15 at 10 28 07 am" src="https://user-images.githubusercontent.com/4295388/32844597-dc670c34-c9f0-11e7-86c5-ab6668f20632.png">


where the json response looks like this:
```json
{
	"count": 4,
	"total": 4,
	"edit": "S205",
	"rows": [{
		"row": { "rowId": "00" },
		"fields": {}
	}, {
		"row": { "rowId": "000" },
		"fields": {}
	}, {
		"row": { "rowId": "000000000000" },
		"fields": {}
	}, {
		"row": { "rowId": "" },
		"fields": {}
	}],
	"_links": { ... }
}
```
